trillian/testdata/int-ca.cert

```diff
@@ -1,14 +1,14 @@
 -----BEGIN CERTIFICATE-----
-MIICGzCCAcKgAwIBAgIEQkJCQjAKBggqhkjOPQQDAjBxMQswCQYDVQQGEwJHQjEP
+MIICHTCCAcKgAwIBAgIEQkJCQjAKBggqhkjOPQQDAjBxMQswCQYDVQQGEwJHQjEP
```

Comment:
This change updates a lot of test files. It was hard to tel if we're testing all the old cases + new ones?

Reply:
Yeah, all the test data gets regenerated but it's mechanical from the `Makefile`.

(It would probably be / have been better to have the synthetic certs entirely generated in code rather than read from a known `../testdata` directory, but that's a separate question/PR).

Reply:
(After in-person discussion I now understand what you're asking)

This change does alter the integration test from testing (chains with no intermediate path-len constraints) to testing (chains with intermediate path-len=0) -- but I don't think that losing the former scenario is a significant loss.

Reply:
Ack.